engine/link/link.asm

```diff
@@ -838,7 +838,8 @@ Link_PrepPartyData_Gen2:
 	ld [de], a
 	ret
 
-Function285d3:
+Link_Fill5Bytes:
```

Comment:
This is only called once; hopefully we can identify what these bytes are for.

Reply:
That single call fills in some padding before the mail data. I can only assume it's equivalent to the serial preamble.